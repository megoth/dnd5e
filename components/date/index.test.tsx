import React from "react";
import { render } from "@testing-library/react";
import DateComponent from "./index";

describe("Date", () => {
  it("renders a date", () => {
    const date = new Date(2014, 6, 3);
    const { asFragment } = render(
      <DateComponent dateString={date.toISOString()} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
