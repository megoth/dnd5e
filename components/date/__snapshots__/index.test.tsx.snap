// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date renders a date 1`] = `
<DocumentFragment>
  <time
    datetime="2014-07-02T22:00:00.000Z"
  >
    July 3, 2014
  </time>
</DocumentFragment>
`;
